contract-examples/test/README.md

```diff
@@ -37,7 +37,7 @@ test(
 Many contract functions can be called as a part of the same test:
 
 ```ts
-test("<test_name>", ["<test_fn1>", "<test_fn2>", "<test_fn3>"])
+test("<step_name>", ["<step_fn1>", "<step_fn2>", "<step_fn3>"])
```

Comment:
should we do:
```suggestion
test("<test_name>", ["<step_fn1>", "<step_fn2>", "<step_fn3>"])
```
?
Also if each step is a transaction maybe that can be reflected in the name somehow?

Reply:
Global search and replace failed me 😞.

>Also if each step is a transaction maybe that can be reflected in the name somehow?

I thought the same thing... couldn't think of anything that didn't look weird though. I think it's fine as is. One really needs to look at the TS to understand what's going on and I think it's clear that `contract.some_method()` is a transaction from any context outside of solidity. And I guess there's nothing preventing someone from defining the following in solidity:
```solidity
function step_1a() public {
    // ...
}
function step_1b() public {
    // ...
}

function step_1() public {
    this.step_1a();
    this.step_1b();
}
```

Though I wouldn't encourage it. 

If you have any ideas that aren't `tx_step_`, I'm open...